vllm_ascend/ops/fused_moe/fused_moe.py

```diff
@@ -292,7 +292,7 @@ def maybe_all_reduce_tensor_model_parallel(
         return torch.ops.vllm.maybe_all_reduce_tensor_model_parallel(
             final_hidden_states)
 
-    def forward_impl(self, hidden_states: torch.Tensor,
+    def forward_native(self, hidden_states: torch.Tensor,
```

Comment:
if we just want to replace the whole forward pipeline of FusedMoE, I think we can just overwirte the `forward_oot` to make it clearly.
```suggestion
    def forward_oot(self, hidden_states: torch.Tensor,
```

Reply:
maybe override `forward` is better